brokerapi/brokers/bigquery/broker.go

```diff
@@ -105,7 +107,7 @@ func (b *BigQueryBroker) Deprovision(instanceID string, details models.Deprovisi
 
 	dataset := models.ServiceInstanceDetails{}
 	if err = db_service.DbConnection.Where("ID = ?", instanceID).First(&dataset).Error; err != nil {
-		return models.ErrInstanceDoesNotExist
+		return brokerapi.ErrInstanceDoesNotExist
```

Comment:
I wonder if there are any callers that do a type assertion on the old type?

Reply:
Behind the scenes the API translates these into HTTP response codes and error messages so we should be alright. The strings should be the same as well because we originally copy/pasted their code and they've only refactored that section only slightly since.